### PR TITLE
docs: added Edge troubleshooting steps

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds an Edge section in the troubleshooting page.

## Review Changes

💻 [Preview](https://deploy-preview-1213--docs-spectrocloud.netlify.app/troubleshooting/edge)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1213#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/14-troubleshooting/06-nodes.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/06-nodes.md:12]:
```suggestion
This page covers common debugging scenarios for nodes and clusters after they have been deployed.
```

Comment (ritawatson) [reply content/docs/14-troubleshooting/06-nodes.md:12]: Here, it's correct to exclude the hyphen because there's nothing after post deployment to modify, but the suggested change might be preferable. 

For example, a hyphen is correct in step 6 because 'read-only' modifies 'file system':

"Reboot the system to resume the default read-only file system."

File: content/docs/14-troubleshooting/06-nodes.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/06-nodes.md:18]:
```suggestion
Palette performs a rolling upgrade on nodes when it detects a change in the `kubeadm` config. Below are some actions that cause the `kubeadm` configuration to change and result in nodes being upgraded:
```

File: content/docs/14-troubleshooting/12-edge.mdx
Comment (ritawatson) [inline content/docs/14-troubleshooting/12-edge.mdx:32]:
```suggestion
![The grubmenu displays with the command rd.cos.debugrw typed in the terminal.](/troubleshooting_edge_grub-menu.png)
```

**Review by ritawatson — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1213#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1213#pullrequestreview-1361855226" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by rishi-anand — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1213#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1213#pullrequestreview-1361894094" title="Approved">rishi-anand</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1213#pullrequestreview-1361855226" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
